ui/src/main/js/page/certificates/DeleteCertificatesModal.js

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { fetchCertificates, deleteCertificate } from 'store/actions/certificates';
+import Modal from 'common/component/modal/Modal';
+
+const useStyles = createUseStyles({
+    deleteConfirmMessage: {
+        margin: [0, 0, '20px', '30px'],
+        fontSize: '14px',
+        fontWeight: 'bold'
+    },
+    cardContainer: {
+        display: 'flex',
+        marginLeft: '50px'
+    },
+    certCard: {
+        display: 'flex',
+        border: ['1px', 'solid', '#D9D9D9'],
+        borderRadius: '5px',
+        backgroundColor: '#e8e6e6',
+        padding: '8px',
+        margin: [0, '50px', '10px', '20px'],
+        width: '250px'
+    },
+    certIcon: {
+        flexBasis: '20%',
+        backgroundColor: 'white',
+        border: ['1px', 'solid', '#D9D9D9'],
+        borderRadius: '5px',
+        height: '50px',
+        paddingTop: '5px',
+        textAlign: 'center'
+    },
+    certInfo: {
+        flexGrow: 1,
+        alignSelf: 'center',
+        margin: [0, '15px']
+    }
+});
+
+
+const DeleteCertificatesModal = ({ isOpen, toggleModal, data, selected }) => {
```

Comment:
Define PropTypes for validations.

Reply:
Addressed in my most recent commit, 43792b148adc1a01681d6bcca568e71d6bc31d8